thanos/README.md

```diff
@@ -177,6 +212,14 @@ This would enable you to either:
           - thanos-kube-2:10901
     ```
 
+### Disable Individual Thanos Components
+It is possible to disable certain Thanos components for your deployment. To do so add the necessary configuration to your Plugin (currently it is not possible to disable the query component)
+```yaml
+- name: thanos.store.enabled
+  value: false
+- name: thanos.compactor.enabled
+  value: false
```

Comment:
Maybe we provide a table here?

| Thanos Component | enabled by default | deactivatable | flag |
| -- | -- | -- | -- | 
| Query | Yes | No | thanos.xyz.enabled |